frontend/src/scenes/authentication/Login.scss

```diff
@@ -4,6 +4,7 @@
 .bridge-page.login {
     .helper-links {
         text-align: center;
+        padding-top: 1rem;
```

Comment:
we're already importing the `vars.scss` in this file so could replace `1rem` with `$default_spacing`

Reply:
adding another commit should also fix the linting error on the PR title :)